### PR TITLE
Random

### PR DESCRIPTION
**Pull request by miigotu**
Fixes filterShows on poster view
Fixes some issues with sorting by name without sort articles enabled.

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked
- [x] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

**Comment by miigotu** [comment]
https://github.com/SickRage/sickrage-issues/issues/395 (and more)

File: gui/slick/js/core.js [hunk 1]
```diff
@@ -1805,8 +1805,8 @@ var SICKRAGE = {
             $('#filterShowName').on('input', _.debounce(function() {
                 $('.show-grid').isotope({
                     filter: function () {
-                      var name = $(this).data('name');
-                      return name.toLowerCase();
+                      var name = $(this).attr('data-name').toLowerCase();
+                      return name.indexOf($('#filterShowName').val()) > -1;
```
Comment (OmgImAlexis) [inline gui/slick/js/core.js:1809]: This shouldn't have been changed, it was set to filter alphabetically via the name returned?

Comment (miigotu) [inline gui/slick/js/core.js:1809]: It did not work at all before this change, it did absolutely nothing. It works now. Its not a filter but a search box. So if you type 'rones' its gonna have "Game of Thrones" in the results. This is different than the filter row.

**Pull request merged**